fv3core/.jenkins/actions/run_regression_tests.sh

```diff
@@ -15,5 +15,9 @@ if [ ${python_env} == "virtualenv" ]; then
     CONTAINER_CMD="srun" make tests savepoint_tests
 else
     export TEST_ARGS="${TEST_ARGS} --junitxml=/.jenkins/${XML_REPORT}"
-    VOLUMES="-v ${pwd}/.jenkins:/.jenkins" make tests savepoint_tests
+    if [[ ${FV3_DACEMODE} == "True" ]]; then
+        VOLUMES="-v ${SCRIPT_DIR}/../:/.jenkins" RUN_FLAGS="--rm -e FV3_DACEMODE=True" make savepoint_tests
+    else
+        VOLUMES="-v ${SCRIPT_DIR}/../:/.jenkins" make tests savepoint_tests
```

Comment:
Actually, unless there's a more elegant way of doing this, we want to run only `savepoint_tests` when `FV3_DACEMODE=True`. I think we could keep RUN_FALGS here instead of appending in the Makefile, because the work flow would have been similar:
```
export FV3_DACEMODE=True
cd fv3core
.jenkins/jenkins.sh run_regression_tests gtc:dace c12_6ranks_standard
```

vs.

```
export FV3_DACEMODE=True
cd fv3core
make savepoint_tests "TEST_ARGS=--backend=gtc:dace"
```

Reply:
I normally run tests using make inside the fv3core directory, and ideally would like to continue doing this. Having the logic inside the Makefile means the same logic gets re-used for make targets and jenkins actions.

I usually expect calling `jenkins.sh` to fail when running on my local machine. For example, `./.jenkins/jenkins.sh run_regression_tests numpy c12_6ranks_standard` crashes with a "slave is not defined" error, for me.

I think it's fine if `FV3_DACEMODE=True make test` is expected to fail, your if-conditionals in the actions scripts avoid doing this.

Reply:
You can still keep the if-statement to toggle whether we're running the `main` tests, if that stays here.

Reply:
If you can, please append (instead of set default for) `-e FV3_DACEMODE=True`, so we can still pass manual RUN_FLAGS without copy-pasting or remembering this flag.

Reply:
Appending `RUN_FLAGS` and passed in Makefile.